arches/app/src/components/ControlledListManager/ControlledListTable.vue

```diff
@@ -0,0 +1,397 @@
+    // eslint-disable-next-line vue/no-mutating-props
+    props.displayedList.items.sort((a, b) => {
+        const indexInDragDataA = dragData.value.findIndex(
+            (item) => item.id === a.id
+        );
+        const indexInDragDataB = dragData.value.findIndex(
+            (item) => item.id === b.id
+        );
+        if (indexInDragDataA === -1 || indexInDragDataB === -1) {
+            return 0;
+        }
+        if (indexInDragDataA < indexInDragDataB) {
+            return -1;
+        } else if (indexInDragDataB > indexInDragDataA) {
+            return 1;
+        }
+        return 0;
+    });
+    for (let i = 0; i < props.displayedList.items.length; i++) {
+        // eslint-disable-next-line vue/no-mutating-props
+        props.displayedList.items[i].sortorder = i;
+    }
+
+    postListToServer(props.displayedList, toast, $gettext);
+};
+
+const itemsForLanguage = computed(() => {
+    // Show/hide rows based on row expansion toggle
+    if (!props.displayedList) {
+        return [];
+    }
+    const itemsToShow = props.displayedList.items.reduce((acc, row) => {
+        if (!row.parent_id) {
+            acc.push(row);
+        } else if (expandedRows.value[row.parent_id]) {
+            acc.push(row);
+        }
+        return acc;
+    }, []);
+
+    // Flatten labels
+    return itemsToShow.map((item) => {
+        return {
+            ...item,
+            prefLabels: [
+                ...item.labels
+                    .filter(
+                        (label) =>
+                            label.language === selectedLanguage.value &&
+                            label.valuetype === "prefLabel"
+                    )
+                    .map((label) => label.value),
+            ].join(", "),
+            altLabels: [
+                ...item.labels
+                    .filter(
+                        (label) =>
+                            label.language === selectedLanguage.value &&
+                            label.valuetype === "altLabel"
+                    )
+                    .map((label) => label.value),
+            ].join(", "),
+        };
+    });
+});
+</script>
+
+<template>
+    <ListHeader
+        :displayed-list="props.displayedList"
+        :is-item-editor="false"
+    />
+
+    <div
+        v-if="props.displayedList"
+        class="list-editor-container"
+    >
+        <ListCharacteristics
+            :displayed-list="props.displayedList"
+            :editable="false"
+        />
+        <div
+            class="items"
+            style="height: 50vh"
+        >
+            <h3 style="margin-top: 4rem; margin-left: 0">
+                Items ({{ props.displayedList.items.length }})
+            </h3>
+            <div style="height: 100%">
+                <div class="controls">
+                    <SplitButton
+                        class="button language-selector"
+                        :label="`Language - ${languageMap?.[selectedLanguage]}`"
```

Comment:
needs i18n

Reply:
+1. I think this button is going away in the next iteration, so will hold off for now.